tests/components/material.test.js

```diff
@@ -173,13 +173,16 @@ suite('material', function () {
       var textureLoaderSpy = this.sinon.spy(THREE.TextureLoader.prototype, 'load');
       img.setAttribute('src', IMG_SRC);
       img.setAttribute('id', 'foo');
-      THREE.Cache.files[IMG_SRC] = img;
```

Comment:
Note that setting the cache in the test was wrong in my opinion, that didn't test that we set the cache for img added in a-assets. I added a comment in the changes.